tests/llm/test_investigate.py

```
@@ -134,67 +121,33 @@ def test_investigate(experiment_name, test_case):
-            scores["sections"] = sections_eval.score
-            sections_span.log(
-                scores={
-                    "sections": sections_eval.score,
-                },
-                metadata=sections_eval.metadata,
-            )
+        sections = {
+            key: bool(value) for key, value in test_case.expected_sections.items()
+        }
+        sections_eval = evaluate_sections(
+            sections=sections, output=output, parent_span=eval_span
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)